tornadis/pool.py

```diff
@@ -88,6 +91,10 @@ def get_connected_client(self):
                 LOG.warning("can't connect to %s", client.title)
                 raise tornado.gen.Return(ClientError("can't connect to %s" %
                                                      client.title))
+
+            if self.db != 0:
+                yield client.call('SELECT', self.db)
```

Comment:
can you add a test about the result of the SELECT command and raise a tornado.gen.Return(ClientError("something")) if there is a problem here ?